Sources/StreamChat/Repositories/PollsRepository.swift

```diff
@@ -144,12 +144,18 @@ class PollsRepository {
         voteId: String,
         completion: ((Error?) -> Void)? = nil
     ) {
-        var pollVote: PollVote?
+        var exists = false
+        var answerText: String?
+        var optionId: String?
+        var userId: UserId?
         var filterHash: String?
         database.write { session in
             let voteDto = try session.removePollVote(with: voteId, pollId: pollId)
+            exists = voteDto != nil
             filterHash = voteDto?.queries?.first?.filterHash
-            pollVote = try voteDto?.asModel()
+            answerText = voteDto?.answerText
+            optionId = voteDto?.optionId
+            userId = voteDto?.user?.id
```

Comment:
Skipped `asModel()` here because now it requires the DTO to not be in the deleted state.

Reply:
Can you clarify this part a bit? I did not fully understood it

Reply:
Ah nevermind got it 👍